src/pcapi/notifications/push/transactional_notifications.py

```diff
@@ -49,15 +50,17 @@ def get_bookings_cancellation_notification_data(booking_ids: list[int]) -> Optio
 
 
 def get_tomorrow_stock_notification_data(stock_id: int) -> Optional[TransactionalNotificationData]:
-    stock = Stock.query.filter_by(id=stock_id).join(Booking).join(Stock.offer).one()
-    bookings = [booking for booking in stock.bookings if not booking.isCancelled]
+    stock = Stock.query.filter_by(id=stock_id).one()
+    individual_bookings = (
+        IndividualBooking.query.join(Booking, Booking.stockId == stock_id).filter(Booking.isCancelled.is_(False)).all()
```

Comment:
```suggestion
        IndividualBooking.query.join(Booking, Booking.stockId == stock_id).filter(Booking.status==BookingStatus.CANCELLED).all()
```

Comme ça on commence à migrer vers le nouveau comportement, non ?

Reply:
Okay, je vis checker avant en base staging si on a bien les statuts et le booléen qui concordent car je ne suis pas sûr (il me semble que c'est une étape qu'on avait laissé pour plus tard)

Reply:
`SELECT * FROM booking WHERE ("isCancelled" is True AND status != 'CANCELLED') OR ("isCancelled" IS False AND status = 'CANCELLED')` => c'est bon, aucun!